threat_intel/util/http.py

```diff
@@ -388,6 +400,8 @@ def _convert_to_json(self, response):
         Returns:
             Response in JSON format if the response can be converted to JSON. `None` otherwise.
         """
+        if 'content-disposition' in response.headers:
+            return self._handle_file_download(response)
```

Comment:
I'm pretty sure I wrote this comment already but github must have swallowed it.
This may break other API clients in case their response contains this header. Since we know in advance which requests return file data, I'd advise to just propagate a bool flag down the `multi_get` function call to indicate we expect a file in the response.